proposals/description/deprecate_incentives.ts

```diff
@@ -0,0 +1,212 @@
+    },
+    {
+      target: 'stakingTokenWrapperNEARLaaS',
+      values: '0',
+      method: 'harvest()',
+      arguments: (addresses) => [],
+      description: 'Harvest the LaaS: NEAR staking token wrapper to fund deposit'
+    },
+    {
+      target: 'stakingTokenWrapperKYLINLaaS',
+      values: '0',
+      method: 'harvest()',
+      arguments: (addresses) => [],
+      description: 'Harvest the LaaS: KYLIN staking token wrapper to fund deposit'
+    },
+    {
+      target: 'stakingTokenWrapperMStableLaaS',
+      values: '0',
+      method: 'harvest()',
+      arguments: (addresses) => [],
+      description: 'Harvest the LaaS: MStable staking token wrapper to fund deposit'
+    },
+    {
+      target: 'stakingTokenWrapperPoolTogetherLaaS',
+      values: '0',
+      method: 'harvest()',
+      arguments: (addresses) => [],
+      description: 'Harvest the LaaS: PoolTogether staking token wrapper to fund deposit'
+    },
+    {
+      target: 'stakingTokenWrapperBribeD3pool',
+      values: '0',
+      method: 'harvest()',
+      arguments: (addresses) => [],
+      description: 'Harvest the Votium bribes: d3pool staking token wrapper to fund deposit'
+    },
+    {
+      target: 'd3StakingTokenWrapper',
+      values: '0',
+      method: 'harvest()',
+      arguments: (addresses) => [],
+      description: 'Harvest the FeiRari: d3Pool LP staking token wrapper to fund deposit'
+    },
+    {
+      target: 'fei3CrvStakingtokenWrapper',
+      values: '0',
+      method: 'harvest()',
+      arguments: (addresses) => [],
+      description: 'Harvest the FeiRari: 3crv-FEI metapool LP staking token wrapper to fund deposit'
+    },
+    {
+      target: 'feiDaiStakingTokenWrapper',
+      values: '0',
+      method: 'harvest()',
+      arguments: (addresses) => [],
+      description: 'Harvest the FeiRari: G-UNI DAI/FEI 0.05% fee tier staking token wrapper to fund deposit'
+    },
+    {
+      target: 'feiUsdcStakingTokenWrapper',
+      values: '0',
+      method: 'harvest()',
+      arguments: (addresses) => [],
+      description: 'Harvest the FeiRari: G-UNI USDC/FEI 0.01% fee tier staking token wrapper to fund deposit'
+    },
+    {
+      target: 'stakingTokenWrapperBribe3Crvpool',
+      values: '0',
+      method: 'harvest()',
+      arguments: (addresses) => [],
+      description: `
+      Harvest the Votium bribes: 3crv-FEI metapool staking token wrapper to fund deposit. This will
+      harvest 1.5M TRIBE and send it to the votiumBriber contract for the 3Crv-pool. This will then later be harvested.
+
+      Harvested here to remove any possibility that the harvest is conducted during the DAO vote and the withdrawal amount is 
+      incorrect. Later withdrawn from the VotiumBriber contract.
+      `
+    },
+    ////////////   END OF PERFORM OUTSIDE OF PROPOSAL FIRST  ///////////////////
+
+    // Withdraw excess TRIBE from reward system
+    {
+      target: 'erc20Dripper',
```

Comment:
This has a permissionless `drip()` function, which is available to be called. It is on a timer of 7 days. We should pause the dripper before this proposal goes up to prevent the balance available to be withdrawn from changing

Reply:
Good spot, checked, you're correct. Dripping will fail